zapalki/src/console_input/PutCommand.java

```diff
@@ -0,0 +1,100 @@
+package console_input;
+
+import resources.Cost;
+import zapalki.Match;
+import zapalki.MatchBox;
+
+import java.util.*;
+import java.util.stream.Collectors;
+
+public class PutCommand implements Command{
+    private static final String NAME = "put";
+    private final MatchBox type;
+    private final int amount;
+
+
+    public PutCommand(MatchBox type,int amount) {
+        this.type = type;
+        this.amount = amount;
+    }
+
+    //put <box type[small,large]> <amount> - to put random matches to a box of given size
+    public static boolean isParsingPossible(String[] cmdParts) {
+        return (cmdParts[0].equals(PutCommand.NAME) && cmdParts.length == 3);
+    }
+
+    public static Command parse(String[] cmdParts) {
+        String boxType = cmdParts[1].toUpperCase();
+        MatchBox type;
+        Cost getMatchBoxType; //cost containes matchbox objects
+        int amount;
+        try {
+            amount = Integer.parseInt(cmdParts[2]);
+        } catch (NumberFormatException nfe) {
+            System.out.println("ERROR: '" + cmdParts[2] + "' is not a number");
+            return null;
+        }
+        Optional<Cost> boxOpt = Arrays.stream(Cost.values())
+                .filter(item -> item.name().equals(boxType))
```

Comment:
This could use the "SMALL" and "LARGE" instead of "SMALL_MATCHBOX" and "LARGE_MATCHBOX"